examples/mnist/simple_model_training.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Simple Model Training Example
+# 
+# This example is the API example for this Ludwig command line example
+# (https://uber.github.io/ludwig/examples/#image-classification-mnist).
+
+# Import required libraries
+
+from ludwig.api import LudwigModel
+import logging
+import shutil
+
+
+# clean out prior results
+try:
+    shutil.rmtree('./results')
+except:
+    pass
+
+
+# set up Python dictionary to hold model training parameters
+model_definition = {...}
```

Comment:
here you can load the YAML with `yaml.safe_load(path)` and then use the loaded dict when initializing the model.

Reply:
made the requested change